examples/01-http.php

```diff
@@ -24,10 +24,8 @@
     $queue($url)->then(
         function (Psr\Http\Message\ResponseInterface $response) use ($url) {
             echo $url . ' has ' . $response->getBody()->getSize() . ' bytes' . PHP_EOL;
-        },
-        function (Exception $e) use ($url) {
-            echo $url . ' failed: ' . $e->getMessage() . PHP_EOL;
+        }, function (Exception $e) {
+            echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Comment:
> Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line.
>
> – https://www.php-fig.org/psr/psr-12/#47-method-and-function-calls

Nit: It looks like PSR-12 suggests each argument in an argument list that is split across multiple lines needs to be a separate line:

```suggestion
        },
        function (Exception $e) {
            echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Note that this only affects function calls where the first argument is on a separate line and not the other examples where the first argument is a multi-line closure.

Reply:
Thank you for informing me about this requirement for PSR-12 @clue. I added the changes so this should be fixed now. 👍